src/libraries/System.Text.RegularExpressions/tests/Regex.Groups.Tests.cs

```diff
@@ -730,6 +730,8 @@ public static IEnumerable<object[]> Groups_Basic_TestData()
             yield return new object[] { null, @"(?:a{2}?){3}?", "aaaaaaaaa", RegexOptions.None, new string[] { "aaaaaa" } };
             yield return new object[] { null, @"(?:(?:[ab]c[de]f){3}){2}", "acdfbcdfacefbcefbcefbcdfacdef", RegexOptions.None, new string[] { "acdfbcdfacefbcefbcefbcdf" } };
             yield return new object[] { null, @"(?:(?:[ab]c[de]f){3}hello){2}", "aaaaaacdfbcdfacefhellobcefbcefbcdfhellooooo", RegexOptions.None, new string[] { "acdfbcdfacefhellobcefbcefbcdfhello" } };
+            // Nested atomic
```

Comment:
A nested atomic to me means one atomic group inside another ? Like your `[InlineData("(?>(?>(?>(?:abc)*)))", "(?:abc)*")]`

Reply:
I meant things nested inside an atomic that could/should themselves be made atomic.